lib/resque_spec/matchers.rb

```diff
@@ -185,13 +201,13 @@ def schedule_queue_for(actual)
   chain :at do |timestamp|
     @interval = nil
     @time = timestamp
-    @time_info = "at #{@time}"
+    @time_info = " at #{@time}"
   end
 
   chain :in do |interval|
     @time = nil
     @interval = interval
-    @time_info = "in #{@interval} seconds"
+    @time_info = " in #{@interval} seconds"
```

Comment:
Thanks. I think I see what you say, however, since a comma is added by each element in the array then the resulting failure text goes like the following:

```
expected that Person would have [a, b] scheduled, at 2015-06-08 17:41:44 -0500, but got Person with [Les, Hill] scheduled at 2015-06-08 17:41:44 -0500 instead
```

(Please check that there is an extra comma after "scheduled").
I don't really like having an extra comma there and I think point 1 is valid. but you tell me, I can leave it with the extra comma if that works for you.